### PR TITLE
Setting up testing environment

### PR DESCRIPTION
**Pull request by Rinbo**
PT: https://www.pivotaltracker.com/story/show/161218435

- Add Testing gems
- Install Rspec and Cucumber
- Add coveralls requirement for cucumber and rspec

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 0) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+end
```
Comment (danmalmx) [inline db/schema.rb:5]: remove unnecessary text

File: lib/tasks/cucumber.rake [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  task notes: 'cucumber:annotations_setup'
+rescue LoadError
+  desc 'cucumber rake task not available (cucumber not installed)'
+  task :cucumber do
+    abort 'Cucumber rake task is not available. Be sure to install cucumber as a gem or plugin'
+  end
+end
+
+end
```
Comment (danmalmx) [inline lib/tasks/cucumber.rake:68]: remove unnecessary text

**Comment by tochman** [comment]
About the title "Testing setup"  what does that mean? Are you testing a setup or are you setting up testing?

**Comment by Rinbo** [comment]
> About the title "Testing setup" what does that mean? Are you testing a setup or are you setting up testing?

I changed the title to 'Setting up testing environment'

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**